backend/src/openarchiefbeheer/destruction/models.py

```diff
@@ -295,6 +295,7 @@ class DestructionListAssignee(models.Model):
         choices=ListRole.choices,
         max_length=80,
     )
+    comment = models.TextField(_("comment"), blank=True, null=True)
```

Comment:
I think that it does not make sense to add this field to the Assignee model, there is no context here about what the comment is about.

I would follow the same pattern that we used for the "reassign" operation: https://github.com/maykinmedia/open-archiefbeheer/blob/main/backend/src/openarchiefbeheer/destruction/api/viewsets.py#L288

Here the comment is only present in the serialiser used for reassigning. The comment is then added to the logs, but it is not related to the assignee that reassigns users.

If we want that this field is stored in the database somewhere else than in a log, we need to think a bit more about how we want to do this.